be/src/storage/rowset/rowset.h

```diff
@@ -247,6 +247,44 @@ class Rowset : public std::enable_shared_from_this<Rowset> {
         }
     }
 
+    void release2() {
```

Comment:
Seems release2() and release() can be unified.

Reply:
There is still some difference between `release` and `release2`. `Release2` only close `stale` rowsets while `release` can close rowsets which are not staled ( Not staled rowsets may be loaded into memory again). 
I think on this side we only need to release the rowsets for staled?

Reply:
If release() closes `stale` and `not stale` rowset both, I think it's better.
I think it's a super set.